### PR TITLE
source files list separated by lines

### PR DESCRIPTION
**Pull request by niXman**
subj.

### CODE REVIEW

**Comment by niXman** [comment]
Ooops =)

Please reject all commits.

**Comment by andreygursky** [comment]
Related: https://github.com/neutralinojs/neutralinojs/issues/140

File: README.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ Check this [article](https://medium.com/@shalithasuranga/getting-started-with-ne
 
 To create Neutralino developer environment in your pc, install these tool kits.
 
-- [MinGW Compiler for C++](http://mingw.org/)
+- [MinGW-W64](https://mingw-w64.org/)
```
Comment (shalithasuranga) [inline README.md]: `MinGW-W64` is improved fork os mingw. So how about telling MinGW-W64 for Windows and GNU GCC/G++ for Linux. Your ideas?

**Comment by shalithasuranga** [comment]
This looks good and improve readability of script. Can you remove dist/binary from the PR

**Pull request closed without merging**